exec/java-exec/src/test/java/org/apache/drill/exec/vector/complex/writer/TestJsonNanInf.java

```diff
@@ -331,4 +343,63 @@ public void testInnerJoinWithNaN() throws Exception {
     }
   }
 
+  @Test
+  public void testHashJoinWithNaN() throws Exception {
```

Comment:
When do you reset join options (`JoinTestBase.resetJoinOptions()`)?

Reply:
@arina-ielchiieva added resetting of default options